html-forms/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
```

Comment:
the action attribute is not needed on this particular form, but we will discuss what it does in Q and A